src/core/prototypes/plot_segments/region_2d/base.ts

```diff
@@ -2757,6 +2768,25 @@ export class Region2DConstraintBuilder {
                 ignoreSearch: true,
               }
             ),
+            m.label(strings.objects.plotSegment.packingInContainer, {
+              ignoreSearch: true,
+            }),
+            m.inputBoolean(
+              { property: "sublayout", field: ["packing", "boxedX"] },
+              {
+                type: "checkbox-fill-width",
+                label: strings.objects.plotSegment.packingX,
+                ignoreSearch: false,
+              }
+            ),
+            m.inputBoolean(
+              { property: "sublayout", field: ["packing", "boxedY"] },
+              {
+                type: "checkbox-fill-width",
```

Comment:
Should the type for the boxedY field be "checkbox-fill-height"? If not, we could use "checkbox-fill" instead of "checkbox-fill-width" because it's used for both width and height.

Reply:
Changed type to 'checkbox'